examples/papi-logging/papi-logging.cpp

```diff
@@ -160,7 +163,7 @@ int main(int argc, char *argv[])
         cg::build()
             .with_criteria(
                 gko::stop::Iteration::build().with_max_iters(20u).on(exec),
-                gko::stop::ResidualNormReduction<>::build()
+                gko::stop::ResidualNormReduction<ValueType>::build()
                     .with_reduction_factor(1e-20)
```

Comment:
I don't remember why this is purposefully set high. Maybe we don't even need it anyway?